lib/hyrax/hirmeos/metrics_tracker.rb

```diff
@@ -15,7 +15,27 @@ def submit_to_hirmeos(work)
     client.post_work(resource_to_hirmeos_json(work))
   end
 
+  def submit_files_to_hirmeos(file_set)
+    work_id = file_set.parent_work_ids.first
+    hirmeos_id = get_translator_work_id(work_id)
+    client.post_files(resource_to_update_hash(file_url(file_set), hirmeos_id))
+  end
+
+  def get_translator_work_id(uuid)
+    response = client.get_work(uuid)
+    work_json = JSON.parse(response.body) if response.success?
+    work_json['data'][0]['work']['UUID'] if work_json.present?
```

Comment:
Would 'dig' be better here?  This seems potentially brittle.

Reply:
I'll take a look